src/flows/recommendation_api/coronavirus_flow.py

```diff
@@ -0,0 +1,88 @@
+from typing import List
+
+from prefect import Flow, task
+
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery, OutputType
+from common_tasks.corpus_candidate_set import (
+    create_corpus_candidate_set_record,
+    load_feature_record,
+    feature_group,
+    validate_corpus_items,
+)
+from utils import config
+from utils.flow import get_flow_name, get_interval_schedule
+from api_clients.sqs import put_results, RecommendationCandidate, NewTabFeedID
+
+'''
+Builds Coronovirus candidate set and writes it to SQS and Feature Store
+'''
+
+# todo: switch to generic curated topic flow
+
+
+FLOW_NAME = get_flow_name(__file__)
+
+CORONAVIRUS_CANDIDATE_SET_ID = 'c29fbf48-0093-49d2-ae55-4e01a3cf800e'
+
+CORONAVIRUS_SQL = """
+SELECT
+    s.resolved_id as id,   
+    c.top_domain_name as publisher,
```

Comment:
TL;DR: For now, getting the publisher from `content.top_domain_name` is ok.

I see that the Corpus `publisher` field is not available in Snowplow or Snowflake at the moment. I've created [HS-116](https://getpocket.atlassian.net/browse/HS-116) to request this data.
- `JOIN content as c ON c.content_id = s.content_id` is not guaranteed to exist, because it relies  on an exact URL comparison between the Corpus API and a Parsed item.
- `top_domain_name` is not guaranteed to be accurate, because the Parser sometimes makes mistakes in extracting the domain from URLs (e.g. "co.uk" for bbc.co.uk), but for publisher spreading it is probably accurate enough.